gdx/src/com/badlogic/gdx/utils/compression/lz/InWindow.java

```diff
@@ -28,8 +28,8 @@ public void MoveBlock () {
 		int numBytes = _bufferOffset + _streamPos - offset;
 
 		// check negative offset ????
-		for (int i = 0; i < numBytes; i++)
-			_bufferBase[i] = _bufferBase[offset + i];
+		if (numBytes >= 0)
+			System.arraycopy(_bufferBase, offset + 0, _bufferBase, 0, numBytes);

```

Comment:
Again, some extra `+0`, but I'd probably not change the compression code, because I'd be afraid of breaking something. This is especially true for the `Encoder` below.